docs/src/benchmark.md

```diff
@@ -13,7 +13,11 @@ We select the problems from `PureJuMP` submodule of `OptimizationProblems` conve
 ``` @example ex1
 problems = (MathOptNLPModel(eval(problem)(), name=string(problem)) for problem ∈ filter(x -> x != :PureJuMP, names(OptimizationProblems.PureJuMP)))
 ```
-Obviously, the same could be achieved using `OptimizationProblems.ADNLPProblems`.
+The same can be achieved using `OptimizationProblems.ADNLPProblems` as follows:
+``` @example ex1
+using ADNLPModels
+ad_problems = (eval(problem)() for problem ∈ setdiff(names(OptimizationProblems.ADNLPProblems), [:ADNLPProblems, :clplatea, :clplateb, :clplatec, :fminsrf2]))
```

Comment:
Thanks @farhadrclass ! Can you rebase this and make the following change
`[:ADNLPProblems, :clplatea, :clplateb, :clplatec, :fminsrf2]` -> `[:ADNLPProblems]`

This issue was fixed in #105 .